src/python/WMCore/WorkQueue/Policy/Start/StartPolicyInterface.py

```diff
@@ -139,7 +139,7 @@ def newQueueElement(self, **args):
         for data, sites in viewitems(ele['Inputs']):
             if not sites:
                 # make call to rucio to get list of replicas for this input data
-                locations = self.getDatasetLocations(data, account=self.rucioAcct)
+                locations = self.getDatasetLocations([data], account=self.rucioAcct)
```

Comment:
Is it worth to check data type of `data` before wrapping it up into list data type? Or allow `getDatasetLocations` to accept both a single dataset or list of datasets?